app/src/test/java/com/antosik/benchproject/app/popular/movies/viewModel/PopularMoviesViewModelTest.kt

```diff
@@ -34,46 +28,22 @@ internal class PopularMoviesViewModelTest {
         Movie(10002, "Test2", 6.4, "2022-06-22", "https://image.tmdb.org/t/p/w500/something2.com"),
         Movie(10003, "Test3", 3.4, "2022-06-21", "https://image.tmdb.org/t/p/w500/something3.com")
     )
-    val badResponse =
-        Result.failure<List<MovieModel>>(
-            Throwable("Unable to resolve host \"api.themoviedb.org\": No address associated with hostname")
-        )
     val getPopularMoviesUseCase: GetPopularMoviesUseCase = mockk()
     val popularMoviesFragmentNavigator: PopularMoviesFragmentNavigator = mockk()
 
     @Test
     fun `when getPopularMoviesUseCase called and load data, it should be movie type`() = runTest {
```

Comment:
Probably should be something like "when getPopularMoviesUseCase called and load data should set popularMovies"

Reply:
Ok, I changed it